dmcontent/govuk_frontend.py

```diff
@@ -378,9 +378,12 @@ def dm_multiquestion(
 
             for item in items:
                 if item["value"] in followups:
-                    followup_q = question.get_question(followups[item["value"]][0])
+                    followup_items = []
+                    for followup_id in followups[item["value"]]:
+                        followup_q = question.get_question(followup_id)
+                        followup_items.append(from_question(followup_q, data, errors, is_page_heading=False))
```

Comment:
```suggestion
                    followup_items = [
                        from_question(question.get_question(followup_id), data, errors, is_page_heading=False)
                        for followup_id in followups[item["value"]]
                    ]
```

Your new version is much clearer. Could we use a generator to make it clearer still?